forms-flow-web/src/modules/userDetailReducer.js

```diff
@@ -39,10 +59,21 @@ const user = (state = initialState, action) => {
       return { ...state, lang: action.payload };
     case ACTION_CONSTANTS.SET_SELECT_LANGUAGES:
       localStorage.setItem("languages", JSON.stringify(action.payload));
-       return { ...state, selectLanguages: action.payload };
+      return { ...state, selectLanguages: action.payload };
+    case ACTION_CONSTANTS.ROLE_IDS:
+      return { ...state, roleIds: setUserRolesToObject(action.payload) };
+    case ACTION_CONSTANTS.ACCESS_ADDING:
```

Comment:
Check if designer then only call the access to populate  . Default empty array in case of array or null if object

Reply:
yah it is populate if the user have designer role